script/scan-inside-docker-container.sh

```diff
@@ -3,5 +3,12 @@ set -eu
 
 # TODO set exit code back to 1
 
-wget -q -O - https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
+# TODO - Remove conditional when https://github.com/aquasecurity/trivy/pull/3985 merged
+if [[ "$(uname -m)" == 'armv7l' ]]; then
+  # wget -q -O - https://github.com/aquasecurity/trivy/releases/download/v0.39.0/trivy_0.39.0_Linux-ARM.tar.gz | tar xvz -C /usr/local/bin
+  wget -q -O - https://raw.githubusercontent.com/YOU54F/trivy/patch-1/contrib/install.sh | sh -s -- -b /usr/local/bin
```

Comment:
😆

Reply:
Always gotta be shaving some yaks 😅

Reply:
just dropped this, and only running the scan for the `amd64` image, due to time constraints and they seemed to report the same vulns across all three so far